<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Домашняя страница</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(2,0,36);
background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="1" column="2" colspan="3">
     <widget class="QLabel" name="our_timer">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLabel" name="before_ev_label">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>До ближайшего события осталось:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="delete_event">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: rgb(32, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Удалить событие</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="active_evs_label">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Активные события:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLabel" name="future_evs_label">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Запланированные события:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="our_clock">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(2,0,36);
background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(9,9,121,1) 35%, rgba(0,212,255,1) 100%);</string>
      </property>
      <property name="text">
       <string>21:51</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="cur_time_label">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текущее время:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="add_event_btn">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: rgb(32, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Добавить событие</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QTextBrowser" name="future_evs_text">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2" colspan="3">
     <widget class="QTextBrowser" name="active_evs_text">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="timer_btn">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: rgb(32, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(0, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Таймер</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
